platforms/pegasus/include/motion_control.hpp

```diff
@@ -0,0 +1,131 @@
+// Project includes
+#include "app_conf.hpp"
+#include "pid_ng/PID.hpp"
+#include "platform.hpp"
+#include "platform_engine/PlatformEngine.hpp"
+#include "quadpid_meta_controller/QuadPIDMetaController.hpp"
+
+namespace cogip {
+
+namespace pf {
+
+namespace motion_control {
+
+typedef uint8_t pid_id_t;
+
+/// Servomotors ids
+constexpr auto START_LINE = __LINE__;
+enum class PidEnum: pid_id_t {
+    LINEAR_POSE_PID = 1,
+    ANGULAR_POSE_PID = 2,
+    LINEAR_SPEED_PID = 3,
+    ANGULAR_SPEED_PID = 4
+};
+constexpr auto PID_COUNT = __LINE__ - START_LINE - 3;
+
+constexpr cogip::uartpb::uuid_t pose_uuid = 1534060156;
+constexpr cogip::uartpb::uuid_t pose_reached_uuid = 2736246403;
+constexpr cogip::uartpb::uuid_t start_pose_uuid = 2741980922;
+constexpr cogip::uartpb::uuid_t state_uuid = 3422642571;
+constexpr cogip::uartpb::uuid_t pid_uuid = 4159164681;
+constexpr cogip::uartpb::uuid_t pid_request_uuid = 3438831927;
+
+constexpr uint16_t motion_control_thread_period_ms = 20;
+
+/// @name Robot mechanical properties
+///
+/// To be computed :
+///  - pulse_per_mm             : number of pulses per mm of coding wheel
+///  - wheels_distance          : distance between coding wheels (pulses)
+///  - pulse_per_degree         : number of pulses per degree of coding wheel
+///
+/// must be known :
+///  - wheels_diameter          : coding wheel diameter (mm)
+///  - wheels_distance_mm       : distance between coding wheels (mm)
+///  - wheels_encoder_resolution: number of pulses by turn of coding wheels
+///
+/// Intermediate calculation:
+///  - wheels_perimeter = pi*wheels_diameter
+///  - pulse_per_mm = wheels_encoder_resolution / wheels_perimeter
+///
+///  - wheels_diameter = 60 mm
+///  - wheels_distance_mm = 280 mm
+///  - wheels_encoder_resolution = 2000
+///
+/// @{
+constexpr double robot_width = 330;                 ///< Robot width (mm)
+constexpr double robot_margin = (robot_width / 2);  ///< Point the most far from robot center (mm)
+constexpr double beacon_support_diameter = 70;      ///< Size of the beacon support (a cylinder of 70mm diameter to a cube of 100mm width)
```

Comment:
Is it still used?

Reply:
no has been removed in a further commit
so done